brand.html

```diff
@@ -83,7 +83,7 @@ <h1 class="reybots">Reynolds Reybots</h1>
                 <picture class="georgie_cobbs" style="width: 100%;">
                     <source srcset="img/brand/georgie-cobbs-unsplash.webp" sizes="" type="image/webp">
                     <source srcset="img/brand/georgie-cobbs-unsplash-original.png" type="image/png"> 
-                    <img src="img/brand/full.png" alt="Photo by Georgie Cobbs on Unsplash">
+                    <img src="img/brand/georgie-cobbs-unsplash-original.png" alt="Photo by Georgie Cobbs on Unsplash">
```

Comment:
Thank you for noticing this error with the file confusion, copy pasting code sometimes has it own problems 😅